pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -263,6 +280,37 @@ in
         contents = [ hello cowsay ];
         singularity = finalAttrs.finalPackage;
       };
+      image-saxpy = callPackage
+        ({ singularity-tools, cudaPackages }:
+          singularity-tools.buildImage {
+            name = "saxpy";
+            contents = [ cudaPackages.saxpy ];
+            memSize = 2048;
+            diskSize = 2048;
+            singularity = finalAttrs.finalPackage;
+          })
+        { };
```

Comment:
Forgetting now if it's an issue one the tests being unfree. Will wait for it to fail